index.md

```diff
@@ -5,19 +5,43 @@
 
-* interfacing with Linux
-* be broadly familiar with various fundamental things like services and daemons
+* be completely comfortable using GNU/Linux
+* have a general understanding of the various parts of the OS,
+  e.g. init, filesystems, services, daemons, etc.
 * understand basic networking
-* have a good sense about
-* maintaining system security
+* have a good sense about maintaining system security
 * understand system administration essentials
-* get a practical taste for what's new in the field
+* get a practical taste of what sysadmins do in industry.
 
-Lectures are scheduled for **8-9pm Tuesdays and Thursdays** in the OCF Lab, located at **171 MLK** (MLK basement down the hall from the student store). **Attendance is mandatory with two allowed unexcused abscences** (other conflicts such as exams can be excused).
+Lectures are scheduled for **Tuesdays and Thursdays 8-9pm** the OCF Lab,
+located at **171 MLK** (MLK basement down the hall from the student store).
+**Attendance is mandatory, with two allowed unexcused absences.**
+(other conflicts such as exams are excused).
 
-The primary assignment in this course will be **weekly lab work**. In keeping with the name of the course, labs will be significantly hands-on. You will be **working on real systems**, configuring and fixing things. **Lecture will double as office hours** for students to come in and ask questions about the lab work. This is a **2 unit DeCal**.
+The primary assignment in this course will be **weekly lab work**.
+In keeping with the name of the course, labs will be significantly hands-on.
+You will be **working on real systems**, configuring and fixing things.
+**Lecture will double as office hours** for students to come in and ask
+questions about the lab work. This is a **2 unit DeCal**.
 
-An important skill for system administrators is the **willingness to learn quickly** about new and unfamiliar technologies, so while we expect many students will have a CS background, **the only real prerequisite is a desire to learn**. We assume **no prior experience** and will get started and move quickly - there's a lot of material to cover. Another important skill to learn is patience and perseverence. Don't be discouraged if things don't work: the internet is held together with the programmatic equivalent of duct tape, and the job of a sysadmin is best described as applying new duct tape when the old stuff wears out, which happens a lot.
+An important skill for system administrators is the **willingness to learn
+quickly** about new and unfamiliar technologies, so while we expect many
+students will have a CS background, **the only real prerequisite is a desire
+to learn**. We know potential students will have varied backgrounds in terms
+of CS and sysadmin experience, therefore, we will be teaching this course in
+two tracks: a beginner's track that will focus more on introducing new users
+to GNU/Linux, and an advanced track that will cater to students with more
```

Comment:
or as I've been referring to it recently, GNU+Linux

Reply:
lol